File: Java/GADSPROJECTAAD/app/src/main/java/com/dodemy/gadsproject_aad/model/SkillIQ.java
package com.dodemy.gadsproject_aad.model;

public class SkillIQ {
    private String name;
    private String country;
    private int score;
    private String badgeUrl;

    public SkillIQ() {
    }


    public SkillIQ(String name, String country, int score, String badgeUrl) {
        this.name = name;
        this.country = country;
        this.score = score;
        this.badgeUrl = badgeUrl;
    }

    public int getScore() {
        return score;
    }

    public void setScore(int score) {
        this.score = score;
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }


    public String getBadgeUrl() {
        return badgeUrl;
    }

    public void setBadgeUrl(String badgeUrl) {
        this.badgeUrl = badgeUrl;
    }

}

File: Kotlin/GADSPROJECTAAD/app/src/main/java/com/dodemy/gadsproject_aad/model/SkillIQ.kt
package com.dodemy.gadsproject_aad.model

class SkillIQ {
    var name: String? = null
    var country: String? = null
    var score = 0
    var badgeUrl: String? = null

    constructor() {}
    constructor(name: String?, country: String?, score: Int, badgeUrl: String?) {
        this.name = name
        this.country = country
        this.score = score
        this.badgeUrl = badgeUrl
    }
}